src/components/Modal.js

```diff
@@ -38,33 +38,38 @@ export const Modal = ({
     cry.play();
   };
 
+  const typeColor = types[0]["type"].name;
+
   return (
     <>
-      <div className="backgroundModalWrapper">
+      <div
+        className="backgroundModalWrapper"
+        style={{ backgroundColor: getTypeColors[typeColor] }}>
         <div className="modalDetailsTop">
           <div className="dexNo">{dexNo}</div>
-
           <div className="spriteWrapper">
             <img src={sprite} alt={`${name} sprite`} />
           </div>
-
           <div className="modalCloseButton">
-          {/* close button */}
+            {/* close button */}
             <button
               onClick={() => {
                 setIsOpen(false);
               }}>
-              <img src={closeButton} alt="close modal button"/>
+              <img src={closeButton} alt="close modal button" />
             </button>
           </div>
         </div>
+
         <div className="modalDetailsBottom">
           <div className="nameContainer">
-            <h1 className="modalPokeName">{name}</h1>
-            <button onClick={playPokemonCry} ref={speakerButtonRef}>
-              <img src={crySpeaker} ref={speakerRef} alt="speaker"/>
-            </button>
-            <span>{typeTags}</span>
+            <div className="nameSpeakerContainer">
+              <h1 className="modalPokeName">{name}</h1>
+              <button onClick={playPokemonCry} ref={speakerButtonRef}>
+                <img src={crySpeaker} ref={speakerRef} />
```

Comment:
You had an `alt` attribute before, but you removed it? Is that kewl for yo' team?

Reply:
ooooh good catch! Thanks!